### ISSUE TITLE
sql: raise the visibility of full table scan queries to users

### ISSUE THREAD

**Issue by arjunravinarayan** [cockroachdb/cockroach#20060]
A common way that users shoot themselves in the foot is by unknowingly performing a full table scan (or a full scan over an index because they've incorrectly constructed the index). We should add a way for users to be alerted to queries that are full scans.

One solution is to add a count of full scan queries in the admin UI. Then when users complain about CPU usage/query performance we can ask if that count is non-zero as a first step to triage the problem.

A second solution is to add a SCAN option to our privileges, and document that users should turn this privilege off in production for their OLTP applications. @petermattis mentioned that some databases have this, but digging around, I could not find it as an option in MySQL, Postgresql, or Oracle.

**Comment by a-robinson** [comment]
See also https://github.com/cockroachdb/cockroach/issues/13969

**Comment by dianasaur323** [comment]
yes yes yes! I'm so glad you filed this. This falls under the query analysis tooling that got punted for 1.2. Take a look at MySQL workbench.

**Comment by bdarnell** [comment]
Percona's slow query log and profiling tools have a field for whether a full table scan was used, as well as some [other attributes](https://www.percona.com/doc/percona-server/5.5/diagnostics/slow_extended_55.html#log_slow_filter). We could do the same thing: however we warn for full-scan queries, also warn for queries that have to use on-disk temp storage, etc.

**Comment by awoods187** [comment]
We do now have: 
![image](https://user-images.githubusercontent.com/22278911/60027015-a7f40e00-966a-11e9-9fc9-c3ff9d7f7e9c.png)
It's a little hard to find as it requires you to click into the query and then understand that spans=all means a full table scan. I think we could raise this up a level in the webui. cc @piyush-singh

**Comment by piyush-singh** [comment]
This should be addressed with the pending IA work - we'll have much clearer ways to call users' attentions to specific issues in the UI cc @Annebirzin

**Comment by awoods187** [comment]
This will be folded in the transactions page https://github.com/cockroachdb/cockroach/issues/46429
